tests/unit/backend/utils/field-validation/attachment-validation.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { ObjectId } from 'mongodb'
+
+import { ValidateFieldError } from 'src/app/modules/submission/submission.errors'
+import { ProcessedAttachmentResponse } from 'src/app/modules/submission/submission.types'
+import { validateField } from 'src/app/utils/field-validation/'
+import { BasicField, IAttachmentFieldSchema, IField } from 'src/types'
+
+import {
+  generateDefaultField,
+  generateNewAttachmentResponse,
+} from '../../helpers/generate-form-data'
+
+describe('Attachment validation', () => {
+  const makeField = (
+    fieldId: string,
+    attachmentSize: string,
+    customParams?: Partial<IField>,
+  ): IAttachmentFieldSchema => {
+    return {
+      ...generateDefaultField(BasicField.Attachment, customParams),
```

Comment:
if `generateDefaultField` is not sufficiently flexible for your purposes, would it be possible to adapt it so that it can be used by other tests in future? I know this is a slippery slope argument but I'm afraid that if we start defining our own `makeField` and `makeResponse` functions everywhere, then our Jest tests are going to end up as messy as our Jasmine ones